assets/strings/eng.json

```diff
@@ -39,9 +39,9 @@
     "Morphological_Generation_Help": "Choose a language and generate surface forms from your text using Apertium's morphological generators.",
     "Generate": "Generate",
     "Spell_Checker": "Spell checker",
-    "APY_Sandbox": "APY sandbox",
-    "APY_Sandbox_Help": "Send arbitrary requests",
-    "APY_Request": "APY Request",
+    "APy_Sandbox": "APy sandbox",
+    "APy_Sandbox_Help": "Send arbitrary requests to ",
```

Comment:
Um, how do you suggest I go about it? I initially thought of keeping a `:` or some symbol and then place the endpoint.